### ISSUE TITLE
[Legitimate Site Blocked] aubus.xyz

### ISSUE THREAD

**Issue by busau** [MetaMask/eth-phishing-detect#6300]
https://aubus.xyz/ 

This is my personal domain/site and it's hardly been used at all so I can't see why it is blocked by you????

Can you please remove.

Edit: I found i can get the reason for the block, it is "This domain was blocked for its similarity to auctus.org, a historical phishing target"

WHAT SORT OF STUPID REASON IS THAT?????????????????????? 

My domain name is nothing like that domain. If you're going to use woeful name matching you should at least look at a site manually before blocking it.

Thanks,
(personal info removed)

**Comment by devenblake** [comment]
Duplicate of #6257.
